addons/portal/static/src/js/portal_chatter.js

```diff
@@ -254,6 +257,26 @@ var PortalChatter = publicWidget.Widget.extend({
         var page = $(ev.currentTarget).data('page');
         this._changeCurrentPage(page);
     },
+
+    /**
+     * @private
+     * @returns {Promise}
+     */
+    _onClickUpdateIsInternal: function (ev) {
+        ev.preventDefault();
+
+        var $elem = $(ev.currentTarget);
+        return this._rpc({
+            route: '/mail/update_is_internal',
+            params: {
+                message_id: $elem.data('message-id'),
```

Comment:
Shouldn't the value be inversed here to actually change it? And the new value written on the element at the end of the rpc.

Reply:
Haha. It was a test for the reviewer.

:man_cartwheeling: